prac_05/hex_colours.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404 2020 - Practical 5
+Student Name: Amy Robinson
+Program - Hex Colours
+"""
+
+HEX_COLOURS = {"blueviolet": "#8a2be2", "coral": "#ff7f50",
+               "firebrick": "#b22222", "forestgreen": "#228b22",
+               "hotpink": "#ff69b4", "brown": "#a52a2a",
+               "chocolate": "#d2691e", "floralwhite": "#fffaf0",
+               "navyblue": "#000080", "light": "#eedd82"}
+
```

Comment:
How would the user know what colours are available? maybe printing a list of the dictionary keys.

Reply:
Added, Thank you.